llvm/utils/lit/lit/cl_arguments.py

```diff
@@ -209,6 +204,17 @@ def parse_args():
         action="store_true",
         help="Exit with status zero even if some tests fail",
     )
+    execution_test_time_group = execution_group.add_mutually_exclusive_group()
+    execution_test_time_group.add_argument(
+        "--skip-test-time-recording",
+        help="Do not track elapsed wall time for each test",
+        action="store_true",
+    )
+    execution_test_time_group.add_argument(
+        "--time-tests",
+        help="Track elapsed wall time for each test in a histogram",
```

Comment:
"in a histogram" doesn't feel right: the times aren't tracked in the histogram, they're printed in that histogram (and recorded in a file).

Reply:
Clarified to say "printed in a histogram"